src/error.rs

```diff
@@ -9,16 +9,23 @@ use rusqlite;
 use serde::{Deserialize, Serialize};
 use serde_json;
 use sqlx;
+use std::fmt::{Display, Formatter};
 use std::string;
 use toml;
 
-#[derive(Debug, Deserialize, Display, Error, Serialize)]
+#[derive(Debug, Deserialize, Error, Serialize)]
 #[serde(tag = "type", content = "value")]
```

Comment:
Do we think it's important to have content=value? I like tagging the type, but do we think it helps to have an extra layer of nesting (and bulking up of the serialized representation? (I'm OK skipping it.)

Reply:
I think it is, at least for UI implementors.

Reply:
The type of the response will be from tag=type, no? I'm curious about content=value, which adds a level of nesting but as best I can tell doesn't provide additional information. It's possible I'm wrong about that, though! :)

Reply:
Yes yes, you're completely right. I realized the example I had in mind doesn't make any sense :sweat_smile:  :p
I'll change it :)

Sorry :p